python/hsfs/core/feature_view_api.py

```diff
@@ -358,3 +366,98 @@ def get_models_provenance(
             explicit_provenance.Links.Type.MODEL,
             training_dataset_version=training_dataset_version,
         )
+
+    def enable_feature_logging(
+        self,
+        feature_view_name: str,
+        feature_view_version: int,):
+        _client = client.get_instance()
+        path_params = self._base_path + [
+            feature_view_name,
+            self._VERSION,
+            feature_view_version,
+            self._LOGGING,
+        ]
+        _client._send_request("PUT", path_params, {})
+
+    def pause_feature_logging(
+        self,
+        feature_view_name: str,
+        feature_view_version: int,):
+        _client = client.get_instance()
+        path_params = self._base_path + [
+            feature_view_name,
+            self._VERSION,
+            feature_view_version,
+            self._LOGGING,
+            self._PAUSE_LOGGING,
+        ]
+        return _client._send_request("POST", path_params, {})
+
+    def resume_feature_logging(
+        self,
+        feature_view_name: str,
+        feature_view_version: int,):
+        _client = client.get_instance()
+        path_params = self._base_path + [
+            feature_view_name,
+            self._VERSION,
+            feature_view_version,
+            self._LOGGING,
+            self._RESUME_LOGGING,
+        ]
+        return _client._send_request("POST", path_params, {})
+
+    def materialize_feature_logging(
+        self,
+        feature_view_name: str,
+        feature_view_version: int,):
+        _client = client.get_instance()
+        path_params = self._base_path + [
+            feature_view_name,
+            self._VERSION,
+            feature_view_version,
+            self._LOGGING,
+            self._MATERIALIZE_LOGGING,
+        ]
+        jobs_json = _client._send_request("POST", path_params, {})
+        jobs = []
+        if jobs_json.get("count", 0) > 1:
+            for item in jobs_json["items"]:
+                jobs.append(Job.from_response_json(item))
+        else:
+            jobs.append(Job.from_response_json(jobs_json))
+        return jobs
+
+    def get_feature_logging(
+        self,
+        feature_view_name: str,
+        feature_view_version: int,):
+        _client = client.get_instance()
+        path_params = self._base_path + [
+            feature_view_name,
+            self._VERSION,
+            feature_view_version,
+            self._LOGGING,
+        ]
+        return _client._send_request("GET", path_params, {})
+
+    def delete_feature_log(
```

Comment:
Not sure what this method is for.
Is it deleting the features that have been logged before or just disabling feature logging?
If it's disabling feature logging only, I would call it `disable_feature_logging` since we have `enable_feature_logging`.
If it's deleting the features previously logged, I would call it `delete_feature_logging`.

Reply:
it means deleting the features previously logged, I modified the method description. It is named similar to `read_log`

Reply:
rename the method in `feature_view_engine` for the name in `feature_view` I think Jim prefer not including the `feature` in the name.